src/test/resources/failing/optional/application.conf

```diff
@@ -0,0 +1,10 @@
+modules = [
+  "myModule=test:io.knotx.launcher.FailingVerticle"
+]
+
+config.myModule.options {
+  config {
+    myValueKey = myValue
+  }
+  optional = true
```

Comment:
We have two `optional` parameters:
- in `bootstrap.json` stores configuration file
- in `configuration.conf` file

Maybe we could rename it to `required`, where the default value is true.